src/test/java/reposense/git/GitLogTest.java

```diff
@@ -64,17 +64,69 @@ public void gitLog_includeAllJavaFilesAuthorIgnoreMovedFile_success() {
         Author ignoreMovedFileAuthor = getAlphaAllAliasAuthor();
         ignoreMovedFileAuthor.setIgnoreGlobList(Collections.singletonList("**movedFile.java"));
 
-        String content = GitLog.get(config, ignoreMovedFileAuthor);
-        Assert.assertTrue(TestUtil.compareNumberExpectedCommitsToGitLogLines(6, content));
+        String content = GitLog.getWithFiles(config, ignoreMovedFileAuthor);
+        Assert.assertTrue(TestUtil.compareNumberFilesChanged(4, content));
     }
 
     @Test
     public void gitLog_authorIgnoreAllJavaFiles_success() {
+        config.setFormats(Collections.singletonList(new Format("java")));
         Author ignoreAllJavaFilesAuthor = getAlphaAllAliasAuthor();
         ignoreAllJavaFilesAuthor.setIgnoreGlobList(Collections.singletonList("*.java"));
 
-        String content = GitLog.get(config, ignoreAllJavaFilesAuthor);
-        Assert.assertTrue(TestUtil.compareNumberExpectedCommitsToGitLogLines(2, content));
+        String content = GitLog.getWithFiles(config, ignoreAllJavaFilesAuthor);
+        Assert.assertTrue(TestUtil.compareNumberFilesChanged(0, content));
+    }
+
+    @Test
+    public void gitLog_authorIgnoreValidFilePath_success() {
+        config.setFormats(Collections.singletonList(new Format("java")));
+        Author ignoreValidFileAuthor = getAlphaAllAliasAuthor();
+
+        // Existing file is ignored
+        ignoreValidFileAuthor.setIgnoreGlobList(Collections.singletonList("annotationTest.java"));
+        String content = GitLog.getWithFiles(config, ignoreValidFileAuthor);
+        Assert.assertTrue(TestUtil.compareNumberFilesChanged(5, content));
+
+        // Non-existing files that are valid
+        ignoreValidFileAuthor.setIgnoreGlobList(Collections.singletonList("**collate"));
+        content = GitLog.getWithFiles(config, ignoreValidFileAuthor);
+        Assert.assertTrue(TestUtil.compareNumberFilesChanged(6, content));
```

Comment:
lets make all these test more relevant (show some changes) to the alpha repo. 
https://github.com/reposense/testrepo-Alpha

E.g. **collate is not found in testrepo alpha, prob use **11612 or something.

Reply:
`.attach_pid11612` is not registered in the list of files. Perhaps because it is not in the list of `DEFAULT_FORMATS`.
I will use another example. Actually some of these tests are for files that are not found.